frontend/test/playwright/e2e/all-results-keyboard.spec.ts

```diff
@@ -59,7 +59,7 @@ test.describe("all results grid keyboard accessibility test", () => {
     )
   })
 
-  test.only("should open audio results as links", async ({ page }) => {
+  test("should open audio results as links", async ({ page }) => {
```

Comment:
Oh dear.

We should add an ESLint restricted syntax rule to prevent this from happening.

```json
{
  "selector":  "CallExpression > MemberExpression:matches([object.name=/(test|it|describe)/], [object.object.name=/(test|it|describe)/])[property.name='only']",
  "message": "Do not commit .only"
}
{
  "selector": "CallExpression > MemberExpression[object.object.name='it'][object.property.name='only']",
  "message": "Do not commit .only"
}
```

That appears to work when I test it locally for `*.only`, `*.only.each` and `*.concurrently.only` which are all the iterations I could find on the jest documentation.